src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const temp = {};
+  const result = [];
+  const dateSplit = date.split(fromFormat[3]);
+
+  for (let i = 0; i < 3; i++) {
+    if (fromFormat[i] === 'YY') {
+      if (dateSplit[i] < 30) {
+        dateSplit[i] = 20 + dateSplit[i];
+      } else {
+        dateSplit[i] = 19 + dateSplit[i];
+      }
```

Comment:
Ternary operator will shorten this part a lot)

Reply:
bit if can help to increase readble